src/patternfly/components/Form/examples/Form.md

```diff
@@ -4,6 +4,23 @@ section: components
 cssPrefix: pf-c-form
 ---
 
+## Examples
```

Comment:
Could you let me know why you add another `Example` section here?

Reply:
Removed it